modules/local/findrepeats/main.nf

```diff
@@ -20,8 +20,14 @@ process FIND_REPEATS {
     path("versions.yml"),             emit: versions
 
     script:
+    // Decompress reference if necessary:
+    def decompress_refgenome = refgenome.toString().endsWith(".gz") ? "gunzip -q -f $refgenome" : ""
```

Comment:
Could you add single quotes around refgenome like `'$refgenome'` as extra protection against e.g., spaces in the file name.